.travis.yml

```diff
@@ -2,15 +2,12 @@ sudo: false
 
 language: go
 
-go:
-        - 1.4
-        - 1.5
-        - 1.6
+branches:
+        only:
+                - merkle-tree
```

Comment:
Yeah, I realized that I probably shouldn't have pushed it to master first. Oh well.